lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -3666,9 +3669,13 @@ For ``text`` inputs the following validators are useful:
 
 - ``length``: Check if the length of the value is within a range.
 - ``empty_field``: Check if the string is not empty
-- ``value_in_data_table`` (``value_not_in_data_table``): Check if the value is
+- ``value_in_data_table``: Check if the value is
 contained in a column of a given data table.
 
+Deprecated:
+
+- ``value_not_in_data_table``: Use ``value_in_data_table`` with ``negate="true"``.
+
```

Comment:
I don't think one is better than the other, I don't think we should call this deprecated. We can internally replace the implementation if you think that makes the code more maintainable, but we don't want tool authors to update existing tools with such a change (and they may feel compelled to do that if we call it deprecated).

Reply:
> I don't think one is better than the other, I don't think we should call this deprecated.

OK. Agreed. I will revert this.

Reply:
> We can internally replace the implementation ...

This is already done :)